Frontend/ui-driver/src/Flow.purs

```diff
@@ -486,6 +487,16 @@ onBoardingFlow = do
     GO_TO_ONBOARD_SUBSCRIPTION -> do
       let onBoardingSubscriptionViewCount =  fromMaybe 0 (fromString (getValueToLocalNativeStore ONBOARDING_SUBSCRIPTION_SCREEN_COUNT))
       onBoardingSubscriptionScreenFlow onBoardingSubscriptionViewCount
+    REFERRAL_CODE_SUBMIT state -> do
+      referDriverResponse <- lift $ lift $ Remote.referDriver (makeReferDriverReq state.data.referralCode)
```

Comment:
better use $ instead of ()
`referDriverResponse <- lift $ lift $ Remote.referDriver $ makeReferDriverReq state.data.referralCode`